app/src/main/res/values/strings.xml

```diff
@@ -34,5 +34,7 @@
     <string name="title_event_detail">Event Detail</string>
     <string name="subtitle_organizer">Organizer: %s</string>
     <string name="event_thumbnail">Event Thumbnail</string>
+    <string name="text_empty">Nothing to show here</string>
```

Comment:
data binding for message string - just suggestion :)
otherwise `Nothing to show here` seems good :+1: